File: c/Lista3/Desafio5.c
#include <stdio.h>
#include <stdlib.h>
#include <math.h>

main(){
    
    float numero, potencia, resultado;
    
    printf("Digite o número: ");
    scanf("%f", &numero);
    
    printf("Digite a potencia: ");
    scanf("%f", &potencia);
    
    resultado = pow (numero,potencia);
printf("O resultado da operação é: %3.0f", resultado);
    
}

File: js/lista3/Desafio5.js
const readline = require('readline');
const rl = readline.createInterface({
  input: process.stdin,
  output: process.stdout
}); 

/*
Desafio5- Construa um programa onde o usuário vai entrar com um número e com a potência desse número:
Calcular a potência;

Mostrar o número e o resultado;

Exemplo 3⁴ = 81.
*/

rl.question('Digite o número: ', (numero) => {
    rl.question('Digite a potência: ', (potencia) => {

        let resultado = Math.pow(numero,potencia);
        console.log(`O resultado da operação é: ${resultado}`);
      rl.close();
    });
  });
      
    

